cloudify_kubernetes/tests/test_k8s_client.py

```diff
@@ -60,7 +62,7 @@ def __getattr__(self, name):
             instance._prepare_payload('unknown_attribute', MagicMock())
 
         self.assertEqual(
-            error.exception.message,
+            text_type(error.exception),
```

Comment:
btw each one of those assertRaises+assertEquals can be replaced with a single assertRaisesRegex